tests/transforms/test_batch_params.py

```diff
@@ -27,20 +27,22 @@ def test_simple_circuit(mocker):
 
     @qml.batch_params
     @qml.qnode(dev)
-    def circuit(data, x, weights):
+    def circuit(data, data2, x, weights):
         qml.templates.AmplitudeEmbedding(data, wires=[0, 1, 2], normalize=True)
+        qml.templates.AngleEmbedding(data2, wires=[0, 1, 2])
         qml.RX(x, wires=0)
         qml.RY(0.2, wires=1)
         qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1, 2])
         return qml.probs(wires=[0, 2])
 
-    batch_size = 3
+    batch_size = 5

```

Comment:
How come the batch size was changed?

Reply:
As you can see, at a certain point I had to apply the transpose to the input. This is a detail that I did not realize at the beginning, so the solution was not correct, but no error occurred. This was because the batch size was iqual to the number of qubits. By changing to a number other than 3 we ensure that we can detect this problem in other situations.